0000-template.md

```diff
@@ -1,3 +1,14 @@
+<!---
+Copyright <fill in year> The Rust Project Developers. See the COPYRIGHT
```

Comment:
@joshtriplett hmmm maybe we could switch to a phrase similar to the one that servo uses (with adjustments ofc):

```text
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Generally I'd prefer to be consistent with `rust-lang/rust` headers, so maybe this could be discussed separately? We can change the header later on to whatever we agreed on to use in `rust-lang/rust`.